### ISSUE TITLE
too few arguments to function 'plpgsql_build_datatype'

### ISSUE THREAD

**Issue by pashagolub** [okbob/plpgsql_check#49]
Trying to build against PostgreSQL 11_STABLE:

```
src/stmtwalk.c: In function 'plpgsql_check_stmt':
src/stmtwalk.c:563:26: error: too few arguments to function 'plpgsql_build_datatype'
  563 |        t_var->datatype = plpgsql_build_datatype(result_oid,
      |                          ^~~~~~~~~~~~~~~~~~~~~~
In file included from src/plpgsql_check.h:3,
                 from src/stmtwalk.c:12:
C:/Users/pasha/Code/POSTGR~1/msys64/usr/local/POSTGR~1/include/server/plpgsql.h:1199:22: note: declared here
 1199 | extern PLpgSQL_type *plpgsql_build_datatype(Oid typeOid, int32 typmod,
      |                      ^~~~~~~~~~~~~~~~~~~~~~
make: *** [<builtin>: src/stmtwalk.o] Error 1
```

**Comment by okbob** [comment]
It is known issue - unfortunately Tom changed internal PLpgSQL API in minor release. It should be fixed when stable version will be tagged 11.6. Because almost all users used tagged version, then code is written for tagged version 11.5. The fix in this moment for you is simple a) use 11.5 or b) remove `#if PG_VERSION_NUM > 110005` before  `#define PLPGSQL_BUILD_DATATYPE_4`

The identifier `PLPGSQL_BUILD_DATATYPE_4` should be defined.

**Comment by pashagolub** [comment]
Thanks a lot, Pavel! Great work! Hope to see you soon at one of the conferences!
